version/compatibility.json

```diff
@@ -1,73 +1,242 @@
 {
   "30": [
-    "v1.10.15",
-    "v1.10.16",
-    "v1.10.17"
+    {
```

Comment:
can we use a different file name for this new version? eg compatibility_v2.json? to avoid CLI from breaking. at least until  a couple of CLI releases supporting the new format are made cc @sukantoraymond